lib/board.rb

```diff
@@ -1,33 +1,53 @@
 class Board
-  attr_reader :grid
-
   def initialize
-    @grid = top_row
-    build_grid
-    add_numbers
+    @grid = build_grid
   end
 
   def to_s
-    grid.to_s
+    @grid
```

Comment:
It's still beneficial to access this via a method, even when it's not intended for public use. Using the instance variable moves away from message passing, which couples this method to that variable.

Reply:
hmmm interesting... I see that!